mail_reply_stage/models/mail_message.py

```diff
@@ -0,0 +1,68 @@
+# Copyright 2025 Quartile (https://www.quartile.co)
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl).
+
+from odoo import api, models
+
+
+class MailMessage(models.Model):
+    _inherit = "mail.message"
+
+    @api.model_create_multi
+    def create(self, values_list):
+        messages = super().create(values_list)
+        for message in messages:
+            user = message.author_id.user_ids[:1]
+            if user and user.has_group("base.group_user"):
+                continue
+            if message.subtype_id and message.subtype_id.internal:
+                continue
+            res_model = (
+                self.env["ir.model"]
+                .sudo()
+                .search([("model", "=", message.model)], limit=1)
+            )
+            if not res_model:
+                continue
+            resource = self.env[message.model].browse(message.res_id)
+            config_records = self.env["mail.reply.config"].search(
+                [("model_id", "=", res_model.id)]
+            )
+            matched_config = None
```

Comment:
```suggestion
            matched_config = None
            parent_field_rec = None
```

Reply:
I think there is a possibility of an error occurring when it goes to the `else` block for the first time.